components/TwoUp/styles.css

```diff
@@ -0,0 +1,40 @@
+two-up {
+  display: grid;
+  --split-point: 0;
+}
+
+two-up > * {
+  grid-area: 1/1;
+}
+
+.two-up-handle {
+  touch-action: none;
+  position: relative;
+  width: 10px;
+  background: red;
+  transform: translateX(var(--split-point)) translateX(-50%);
+  will-change: transform;
+}
+
+.two-up-handle::after {
+  content: '';
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 80px;
+  height: 40px;
+  background: red;
+  border-radius: 20px;
+}
+
+two-up > :nth-child(1):not(.two-up-handle) {
```

Comment:
Is the second `:not()` necessary considering the code makes sure that the handle is always the last child?

Reply:
If there are no children it'll be both the last and first child.